### ISSUE TITLE
bug: Issue in production - Meltano container intializing but outputing no logs

### ISSUE THREAD

**Issue by SBurwash** [meltano/meltano#6849]
### Meltano Version

2.7.1

### Python Version

3.8

### Bug scope

Configuration (settings parsing, validation, etc.)

### Operating System

Linux - Ubuntu 20.04

### Description

Good day,

I recently upgrade to meltano 2.7.2 without any other changes to my dockerfile, and am now having issues in production.

Locally, the project build with no issues, but in production (on ECS) we have the following output:

```


04/10/2022, 17:01:25 | [2m2022-10-04T21:01:25.719470Z[0m [[32m[1minfo [0m] [1mEnvironment 'prod' is active[0m | airflow-scheduler
-- | -- | --
04/10/2022, 17:01:25 | [2m2022-10-04T21:01:25.715461Z[0m [[32m[1mdebug [0m] [1mProject is read-only.[0m | airflow-scheduler

```

Where the meltano project opens a connection with RDS but sends no information.

Would you have any insights on what could cause this issue?

### Code

For a representation of our ECS architecture, see here: https://github.com/potloc/meltano-ecs-fargate-terraform-example

**Comment by tayloramurphy** [comment]
Relevant Slack thread https://meltano.slack.com/archives/C01TCRBBJD7/p1664920710879279?thread_ts=1664742346.311869&cid=C01TCRBBJD7

@SBurwash we'll likely need more information to help debug if you're still seeing this. Sanitized meltano.yml and any logs you can share would be great too. cc @aaronsteers

**Comment by SBurwash** [comment]
Hey @tayloramurphy :D 

We talked about this issue [during office hours ](https://youtu.be/2W8Vq1ySgoI?t=2050); it seems that all three instances of meltano were trying to update the DB at the same time, resulting in lock-in.

2.7.1 was a meltano upgrade that included a large DB upgrade, and my containers were all trying to perform this update at the same time.

We concluded during office hours that in the future, it would be nice for meltano to output a message when they are updating the DB.
